### PR TITLE
@releng Update the versions for commerce-cif-connector and core-cif-components in POM

### PR DESCRIPTION
**Pull request by dplaton**
## Description

Update `core-cif-components` to `1.0` and `commerce-cif-connector` to `1.0`

## Related Issue

N/A

## Motivation and Context

Prepare for a new version release

## How Has This Been Tested?

Functional testing

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ The latest version of the AEM CIF Project Archetype, requires the below minimum
 
 | Archetype | AEM 6.4 | AEM 6.5 | Magento                   | CIF Connector | Java |
 | --------- | ------- | ------- | ------------------------- | ------------- | ---- |
-| 0.10.0    | 6.4.4.0 | 6.5.0   | 2.3.4                     | 0.10.0        | 1.8  |
+| 0.10.0    | 6.4.4.0 | 6.5.0   | 2.3.4                     | 1.0.0       | 1.8  |
```
Comment (cjelger) [inline README.md]: Should be `0.11.0` right? And the old stuff should go into the VERSION.md "archive".

Comment (dplaton) [reply README.md]: I'll update this. I wasn't sure what to write here :)

**Review by cjelger — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ The latest version of the AEM CIF Project Archetype, requires the below minimum
 
 | Archetype | AEM 6.4 | AEM 6.5 | Magento                   | CIF Connector | Java |
 | --------- | ------- | ------- | ------------------------- | ------------- | ---- |
-| 0.10.0    | 6.4.4.0 | 6.5.0   | 2.3.4                     | 0.10.0        | 1.8  |
+| 0.11.0    | 6.4.4.0 | 6.5.0   | 2.3.4                     | 1.0.0       | 1.8  |
```
Comment (cjelger) [inline README.md]: I approved too quickly: for the connector, I added the Magento versions: `2.3.4 & 2.3.5` to indicate it also works with Magento 2.3.5

Comment (dplaton) [reply README.md]: Done

**Pull request merged**